compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -616,6 +616,9 @@ public void visit(BLangLiteral literalExpr) {
 
     @Override
     public void visit(BLangConstRef constRef) {
+        if (constRef.symbol != null && setEnclosingNode(constRef.symbol.owner, constRef.pkgAlias.pos)) {
+            return;
+        }
         this.symbolAtCursor = constRef.symbol;
```

Comment:
If 619 fails, then at 622 we will assign a null value?

Reply:
Yes, that would be fine. If `symbolAtCursor` is `null` at the end it'll result in `Optional.empty()` being returned to the user.